src/client/theme-default/components/VPSponsorsGrid.vue

```diff
@@ -1,4 +1,5 @@
 <script setup lang="ts">
+import type { GridSize } from '../composables/sponsor-grid'
```

Comment:
Put this import below `useSponsorsGrid`.

We could consider setting the `import/order` eslint rule